proposals/lip-0039.md

```diff
@@ -137,7 +138,7 @@ class EmptyNode:
         return EMPTY_HASH
 ```
 
-Here the function `hash` returns the SHA-256 hash of the input. Similar to what we do for regular Merkle trees, we use a different hash function for leaf and branch nodes to protect against second preimage attacks and prefix the hashed data with different constants.
+Here the function `sha256` returns the SHA-256 hash of the input. Similar to what we do for regular Merkle trees, we use a different hash function for leaf and branch nodes to protect against second preimage attacks and prefix the hashed data with different constants.
```

Comment:
`sha256` -> `hash`

Reply:
I use sha256 to distinguish the sha hashing function from the hash property of the node